locales/locales.php

```diff
@@ -1846,6 +1879,14 @@ public function __construct() {
 		$sw->slug = 'sw';
 		$sw->google_code = 'sw';
 		$sw->facebook_locale = 'sw_KE';
+		
+		$syr = new GP_Locale();
+		$syr->english_name = 'Syriac';
+		$syr->native_name = 'Syriac';
+		$syr->lang_code_iso_639_3 = 'syr';
+		$syr->country_code = 'IQ';
+		$syr->wp_locale = 'syr';
+		$syr->slug = 'syr';
```

Comment:
Syriac is a macro language so it's a bit complicated here atm, but I've not found a reference to sy_SY anywhere, http://search.language-archives.org/search.html?q=Syriac lists Iraq, France, and Turkey as countries where the language is used. Assyrian Neo-Aramaic, which is one of the sublanguages (https://www.ethnologue.com/language/aii) has Syria and Jordan listed in addition to Iraq, but since we're creating a locale for the macro language, let's use IQ.